cryptofeed/backends/backend.py

```diff
@@ -112,3 +112,26 @@ async def __call__(self, *, feed: str, symbol: str, start: float, stop: float, i
                 'volume': self.numeric_type(volume), 'closed': str(closed)
                 }
         await self.write(feed, symbol, timestamp, receipt_timestamp, data)
+
+
+class BackendOrderInfoCallback:
+    async def __call__(self, *, feed: str, account:Decimal, symbol: str, order_id: str, timestamp: float, side: str, status: str, 
+                                amount: Decimal, filled: Decimal, remaining: Decimal, price: Decimal, clientOrderId: str, postOnly: bool, 
+                                orderType: str, timeInForce: str, lastTradeTimestamp: float, receipt_timestamp=None):
+        data = {'feed': feed, 'account': account, 'symbol': symbol, 'order_id': order_id, 'timestamp': timestamp, 'side': side, 'status': status, 'amount': amount, 'filled': filled, 'remaining': remaining, 
+                'price': price, 'clientOrderId': clientOrderId, 'postOnly': postOnly, 'orderType': orderType, 'timeInForce': timeInForce, 'lastTradeTimestamp': lastTradeTimestamp}
+        await self.write(feed, symbol, timestamp, receipt_timestamp, data)
+
+
+class BackendPositionsCallback:
+    async def __call__(self, *, feed: str, account: Decimal, timestamp: float, symbol: str, initialMargin=None, maintenanceMargin: Decimal, entryPrice=None, homeNotional: Decimal, 
+                                foreignNotional: Decimal, leverage=None, unrealizedPnl: Decimal, contracts: Decimal, liquidationPrice=None, markPrice: Decimal, receipt_timestamp=None):
```

Comment:
Since parameters are passed as a **params, which  is a dictionary of values and keywords that do not match a parameters. So, there is no problem. Here is an example code: 
`class Callback:
  def __call__(self, feed=None, account=None, timestamp=None):
    data = {'feed': feed, 'account': account, 'timestamp': timestamp}
    print(data)

params = {'account': 897812}
cb = Callback()
cb(**params)`

Output: `{'feed': None, 'account': 897812, 'timestamp': None}`